compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -286,8 +286,30 @@ LogicalResult verifyConvTileAndDecomposeExpertConfig(
             owSize = shape[3];
             return success();
           })
+          .Case<linalg::PoolingNhwcSumOp, linalg::PoolingNhwcMaxOp,
```

Comment:
This case uses the identical pattern order to `linalf::Conv2DNhwcHwcfOp`. Just use that case.

Reply:
I am not sure if it's a good idea because they could be different later; but I am changing it for now.